apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -253,6 +253,16 @@ type EnvoyConfig struct {
 	// +optional
 	Service *NamespacedName `json:"service,omitempty"`
 
+	// LoadBalancer specifies how Contour should set the ingress status address.
+	// If provided, the value can be in one of the formats:
+	//   - hostname:<address,...>: Contour will use the provided comma separated list of addresses directly.
```

Comment:
currently we support setting ingress status address via config flag/parameter here and it can be an IP or hostname

would be good to keep that functionality (and not just support hostnames) if we intend to deprecate the old method of configuring this

looks like that might just mean a naming change here and elsewhere since existing helpers are being used